packages/stripe-integration/README.md

```diff
@@ -0,0 +1,11 @@
+# stripe-integration
+
+This library was generated with [Nx](https://nx.dev).
```

Comment:
Can we please add a better readme here.

Reply:
I edited readme file and added some more informations.